### ISSUE TITLE
Jetpack's not protecting from falling if equipped to suit storage slot instead of a backpack one

### ISSUE THREAD

**Issue by HetNeSS** [Aurorastation/Aurora.3#3201]
> You close the maintenance hatch of the power storage unit.
You put the screwdriver into the tool-belt.
You put the multitool into the tool-belt.
You deploy your suit helmet, sealing you off from the world.
You are now running on internals from the void jetpack (oxygen) on your suit.
You toggle the stabilization on.
You toggle the stabilization off.
You toggle the thrusters on.
You toggle the thrusters off.
You toggle the thrusters on.
You toggle the stabilization on.
You fall through the open space!
With a loud thud, you land on the sand!
A gaping wound opens up on the voidsuit!
You feel something rip in your upper body!
Something feels like it shattered in your upper body!

According to alberyk, toggled on jetpack with stabilisation on do not protect from falling into chasms if human is wearing it on suit storage slot. Same with drones, still getting fall damage even while active and working

![bandicam 2017-08-01 04-06-14-204](https://user-images.githubusercontent.com/18286472/28802341-84da6a94-766f-11e7-8e93-3ae7200e7c8d.png)

**Comment by SoundScopes** [comment]
From what I remember, it's intentional that it only works on the back slot. 
Having in suit storage doesn't allow it to work.

**Comment by skull132** [comment]
What Scopes and Lohikar said.

**Comment by HetNeSS** [comment]
So, why it should work only on the back slot?

**Comment by skull132** [comment]
Jetpacks operate on the premise of releasing jets of gas in predetermined directions in predetermined ratios to eachother. They are specifically designed to work on your back, anywhere else, the jets would be positioned aloof and you would not be controllable.

TL;DR: thrust vectors would be bad.

**Comment by HetNeSS** [comment]
And what's about suit storage? Is it supposed to be some kind of internal storage like one who's robot Bender did have within a futurama movie?

**Comment by skull132** [comment]
It's usually unspecified. Though yes, it's probably internal or semi-internal. How or why a jetpack fits in there is unknown, though probably just a quirk of the jetpack inheriting from the oxygen tank.
